articles/connections/passwordless/configure-passwordless-connection.md

```diff
@@ -0,0 +1,137 @@
+
+4. In **Message**, enter the body text of the SMS.
+
+**Make sure that your Message includes `@@password@@` since this is the placeholder that will be replaced with the one-time password that is sent to the user.**
+
+You can use multiple languages in your SMS Messages. You can choose the language used by calling the [`passwordless/start`](/docs/api/authentication/reference#get-code-or-link) endpoint and setting the `x-request-language` header to the language of your choice. Otherwise, the language used will be extracted from the `accept-language` header (which is automatically set by your browser).
+
+The **Message** area accepts Markdown and Liquid syntax. If you choose to use Liquid, you can programmatically construct elements of the message. For example, you can reference the `request_language` parameter to change the language of the message:
+
+```text
+{% if request_language contains 'dutch' %}
+ Hier is uw verificatie code: @@password@@
+{% endif %}
+
+{% if request_language contains 'fr-FR' %}
+ Ceci est votre code: @@password@@
+{% endif %}
+```
+
+The following parameters are available to you when defining the message template (regardless of which syntax you use):
+
+| Parameter | Description |
+| - | - |
+| `password` or `code` | The one-time password to send to the user |
+| `phone_number ` | The user's phone number |
+| `application.name` | The name of the application to which the user is logging in |
+| `request_language` | The requested language to be used for the SMS message's content |
+
+5. Adjust settings for your **OTP Expiry** and **OTP Length**.
+
+| Parameter | Description |
+| - | - |
+| OTP Expiry | The amount of time before the one-time password expires |
+| OTP Length | The length of the one-time password sent to the user |
+
+If you choose to extend the OPT expiry time, we recommend increasing the OTP length for security reasons. Otherwise, the malicious party has a longer time frame to try to guess a short code.
+
+6. Disable Sign Ups
+
+If you want to restrict sign-ups and only allow the addition of new users via the API or the Dashboard, click the **Disable Sign Ups** toggle to enable this feature.
+
+When you are done making configuration changes, click **Save**.
+
+## Configure Passwordless using email
+
+1. Navigate to the [Connections > Passwordless](${manage_url}/#/connections/passwordless) page in the [Auth0 Dashboard](${manage_url}/), and enable the Email toggle.
+
+![Enable Email Passwordless](/media/articles/connections/passwordless/connections-passwordless-list.png)
+
+2. Enter the email address you want users to see as the sender in the **From** field, and set the **Subject** line for your email.
+
+![Configure Email Passwordless](/media/articles/connections/passwordless/connections-passwordless-email.png)
+
+::: note
+If you are using a custom template, you must change the **From** value to an address that does *not* use the domain **auth0.com**. Otherwise, the default email template will be sent.
```

Comment:
I didn't know I could use a custom template yet, so it's jarring to hear about templates when I'm just trying to set my From and Subject lines. Maybe a brief sentence that explains that the From value is tied to the template and briefly say why as the first sentence of this note?